### ISSUE TITLE
Idea (Discussion) - Address notation & DNS contents

### ISSUE THREAD

**Issue by WietseWind** [RossMacFirdeen/WalletDNS#1]
Let's start with: @RossMacFirdeen 👍 Awesome idea! Could even be combined with something like [tagged addresses](https://github.com/xrp-community/standards-drafts/issues/6) to prevent having to store both the destination account and destination tag.

I think the proposal would work best when the notation format:
- Looks like a domain
- Looks like an e-mail address
... but: _is neither_, to prevent mixups.

Eg. something like:

```
family$wietse.com
```

The format looks like an e-mail address, but uses let's say a `$` ("value") instead of an `@`. The DNS contents could be ledger-independent, using a TXT format like SPF, eg.:

```
Query:
nslookup -q=txt family._payments.wietse.com

Response:
Non-authoritative answer:
family._payments.wietse.com    text = "XRPL=rPdvC6ccq8hCdPKSPJkPmyZ4Mi1oG2FFkT"
family._payments.wietse.com    text = "ETH=0xB32ca952459dF115bDf9FC5D9a1A2a2C89E473B9"
```

When used within a specific ecosystem, of course the client would look specifically for the TXT record prefixed with the used ledger name. However, in a multi-ledger environment, the ledger could be added (forced):

```
family+xrpl$wietse.com
```

-- Oh, and did you know webclients could easily use a public DNS query endpoint like offered by Google?
https://dns.google.com/resolve?name=_payments.fudbingo.com&type=TXT

**Comment by nbougalis** [comment]
I agree that this should probably use tagged addresses to simplify things. I like your proposal, but would suggest that the content of the record by a URI-like thing.
